README.md

```diff
@@ -1,23 +1,147 @@
 # :flamingo: flatzingo
 
-A flatzinc frontend for clingcon
+A [FlatZinc](https://www.minizinc.org/doc-2.4.3/en/flattening.html) frontend to solve CP problems in [MiniZinc](https://www.minizinc.org) format using [clingcon](https://potassco.org/clingcon/) as solver.
 
-Basically the following pipeline:
+The process is done in two stages:
+1. Transform files from MiniZinc in the flattened representation FlatZinc using [fzn2lp](https://github.com/potassco/fzn2lp) into `lp` files.
+2. With the encodings in files [encoding](encoding.lp) and [types](types.lp), the `lp` file is proceeded to use the constraints with the syntax of `clingcon`.
```

Comment:
There is also an output module for translating it into MiniZinc output format

Reply:
I dont understand this.

Reply:
There is the fzn-flatzingo.sh script that takes the fzn, translates it to lp, solves it, and afterwards also converts the answer set to flatzinc output.